### DISCUSSION TITLE
Unable to install data cube - error: conflicting dependencies

### DISCUSSION THREAD

**Discussion by halley276** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#66] [Q&A] [answered]
Hello everyone. I am having trouble installing data cube. I am using pip install and have just updated pip, but I get this error message shown below. Could someone help me with this? Thanks so much.

![Screenshot 2021-05-22 at 7 31 17 PM](https://user-images.githubusercontent.com/65350949/119225142-a40de680-bb34-11eb-8cca-fb0f20739089.png)

![Screenshot 2021-05-22 at 7 31 33 PM](https://user-images.githubusercontent.com/65350949/119225146-a7a16d80-bb34-11eb-8447-8400642e8c86.png)

**Comment by EY-glowglowglow** [comment]
@alexgleith  @codeindulgence  can you please help with this error?

**Comment by nickbutler** [comment, marked as answer]
Hi @halley276, The environment is already set up with `datacube` and other packages.
You should be able to `import datacube`, also running `pip show datacube` from a terminal should also confirm its presence.

Thanks!

**Reply by halley276** [reply]
Thanks @codeindulgence, I found that it works well on Azure VM, just not on my local machine
